kale/predict/losses.py

```diff
@@ -36,7 +36,7 @@ def topk_accuracy(output, target, topk=(1,)):
     """Computes the top-k accuracy for the specified values of k.
 
     Args:
-        output (Tensor): output (Tensor): The output of the last layer of the network, before softmax. Shape: (batch_size, class_count).
+        output (Tensor): The output of the last layer of the network, before softmax. Shape: (batch_size, class_count).
         target (Tensor): The ground truth label. Shape: (batch_size)
         topk (tuple(int)): Compute accuracy at top-k for the values of k specified in this parameter.
```

Comment:
k or K (in text)

Reply:
I don't think this is my code. But I see both k (in sklearn: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.top_k_accuracy_score.html) and K (in torchmetric: https://torchmetrics.readthedocs.io/en/v0.9.1/classification/accuracy.html).